### ISSUE TITLE
kernel mi5s and mi5splus

### ISSUE THREAD

**Issue by anetar** [MiCode/Xiaomi_Kernel_OpenSource#86]
Hopefully source version as soon as the suspended updated with other Source  versions I see oneplus company going in the way of adsorbed all its devices is  updated source
very soon released  to kernel sources and camera blobs

https://github.com/OnePlusOSS

I hope Xiaomi to become like oneplus

**Comment by TomiOhl** [comment]
Yes, I'm thinking of buying the device. Although I love MIUI, sometimes I want to flash a another ROM... So kernel sources would be essential

**Comment by blackjetcat** [comment]
Now you can close issue, source released ( scorpio-m-oss ).
